test/gateway/ost_prime/initialize.js

```diff
@@ -0,0 +1,96 @@
+const OSTPrime = artifacts.require("OSTPrime")
+  , BN = require('bn.js');
+
+const Utils = require('../../../test/test_lib/utils');
+
+contract('OSTPrime.sol', function (accounts) {
+
+  describe('initialize', async () => {
+
+    const DECIMAL = new BN(10);
+    const POW = new BN(18);
+    const DECIMAL_FACTOR = DECIMAL.pow(POW);
+    const TOKENS_MAX = new BN(800000000).mul(DECIMAL_FACTOR);
+
+    let brandedTokenAddress, ostPrime;
+
+    beforeEach(async function () {
+      brandedTokenAddress = accounts[2];
+      ostPrime = await OSTPrime.new(brandedTokenAddress);
+    });
+
+    it('The balance of OST prime contract must be zero', async function () {
+
+      let balance = await Utils.getBalance(ostPrime.address);
+      assert.strictEqual(
+        new BN(0).eq(new BN(balance)),
+        true,
+        `The balance of contract must be zero.`
+      );
+
+    });
+
+    it('should fail if the initialize is called with payable amount not equal ' +
+      'to 800M ost prime base token', async function () {
+
+      let tokenAmount = new BN(1);
+
+      await Utils.expectRevert(
+        ostPrime.initialize.call(
+          {from: accounts[2], value:tokenAmount}
+        ),
+        "Payable amount must be equal to total supply of token."
+      );
+
+    });
+
+    it('should pass if the initialize is called with value equal than 800M ' +
+      'ost prime base token', async function () {
+      let result = await ostPrime.initialize.call(
+        {from: accounts[2], value:TOKENS_MAX}
+      );
+
+      assert.strictEqual(
+        result,
+        true,
+        `The result should be true.`
+      );
+
+      await ostPrime.initialize(
+        {from: accounts[2], value:TOKENS_MAX}
+      );
+
+      let balance = await Utils.getBalance(ostPrime.address);
+      assert.strictEqual(
+        TOKENS_MAX.eq(new BN(balance)),
```

Comment:
`BN::eqn` to compare to numbers (non-BN).
```suggestion
        TOKENS_MAX.eqn(balance),
```

Again, `assert()` instead of `assert.strictEqual()`?

Reply:
balance is `string`, so its converted to `BN`. 
This is already using `assert.strictEqual`.
```
let balance = new BN(await Utils.getBalance(ostPrime.address));
    assert.strictEqual(
      TOKENS_MAX.eq(balance),
      true,
      `The balance of contract must be ${TOKENS_MAX}.`,
    );
```